### DISCUSSION TITLE
Using larger models in docker

### DISCUSSION THREAD

**Discussion by EighthOctave** [rsxdalv/TTS-WebUI#138] [Q&A]
I can successfully build and run the docker container (thanks for that by the way!).  I noticed in the logs that the container loads the smaller bark models.  I'd like to use the larger ones since I have the VRAM to support. How do I enable this in the docker container?

I'm a docker novice, BTW.

**Comment by rsxdalv** [comment]
I believe it should work just by using the settings for bark within the webui. Please let me know if it doesn't function properly.
